lifecycle-operator/main.go

```diff
@@ -142,7 +142,9 @@ func main() {
 	keptnMeters := telemetry.SetUpKeptnTaskMeters(meter)
 
 	// Start the prometheus HTTP server and pass the exporter Collector to it
-	go serveMetrics()
+	ch := make(chan int)
+	go serveMetrics(ch)
+	ch <- env.KeptnDoraMetricsPort
```

Comment:
I don't think you need a channel here. You could also just pass the env variable in directly.

Reply:
done